.github/workflows/publish.yml

```
@@ -0,0 +1,52 @@
+        run: bun i
+
+      - name: Build project
+        run: bun run build
+
+      - name: Publish to npm
+        run: bun publish
+        env:
+          NPM_CONFIG_TOKEN: ${{ secrets.NPM_TOKEN }}
+
+      - name: create release
+        uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)